test-parachains/adder/collator/src/main.rs

```diff
@@ -47,6 +47,8 @@ const GENESIS_BODY: AdderBody = AdderBody {
 #[derive(Clone)]
 struct AdderContext {
 	db: Arc<Mutex<HashMap<AdderHead, AdderBody>>>,
+	/// We store it here to make sure that our interfaces require the correct bounds.
```

Comment:
If I understand correctly this is done to make sure our instance of `Network` is `Send + Sync`.

Reply:
Yeah, the code compiled before, but while integrating it into `Cumulus` the compiler denied it :(